tests/test_document.py

```diff
@@ -58,6 +59,19 @@
         await doc2.save()
 
 
+async def test_conflict_without_rev(filled_database):
```

Comment:
```suggestion
async def test_conflict_without_rev(database):
```

The pre`filled_database` fixture isn't needed for that test. The empty `database` fixture is sufficient.

Reply:
Sure!